cmd/server/main.go

```diff
@@ -1,7 +1,69 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+	"net/http"
+	"os"
+
+	"log"
+
+	"github.com/golang-migrate/migrate/v4"
+	_ "github.com/golang-migrate/migrate/v4/database/postgres"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
+	_ "github.com/jackc/pgx/v4/stdlib"
+	"github.com/jmoiron/sqlx"
+)
 
 func main() {
-	fmt.Println("This is where the server goes")
+	db, err := sqlx.Connect("pgx", os.Getenv("POSTGRESQL_URL"))
+	if err != nil {
+		log.Fatalf("Unable to connect to database: %v\n", err)
+		os.Exit(1)
+	}
+	defer db.Close()
+
+	log.Println("Successfully connected to database")
+
+	// just logging error rather than failing here as
+	// a no changes migration is treated as an error..
+	if err := applyMigrations(); err != nil {
```

Comment:
add TODO here to eventually find a way to fail startup if migrations fail? (Don't want the app coming up and having unpredictable behavior / data consistency issues)